hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/ContainerKeyMapperTask.java

```diff
@@ -216,6 +216,10 @@ public Pair<String, Boolean> process(OMUpdateEventBatch events) {
         continue;
       }
       String updatedKey = omdbUpdateEvent.getKey();
+      if (!isOmKeyInfo(omdbUpdateEvent.getValue())) {
```

Comment:
you can combine this if condition with if condition at line 215 to avoid line 218 execution.